dali/core/mm/async_pool_test.cu

```diff
@@ -351,5 +350,103 @@ TEST(MMAsyncPool, CrossStreamWithHogs) {
   upstream.check_leaks();
 }
 
+#if DALI_USE_CUDA_VM_MAP
+
+TEST(MM_VMAsyncPool, MultiThreadedSingleStreamRandom) {
+  if (!cuvm::IsSupported())
+    GTEST_SKIP() << "Virtual memory management API is not supported on this machine.";
+
+  CUDAStream stream = CUDAStream::Create(true);
+  {
+    vector<block> blocks;
+    std::mutex mtx;
+
+    async_pool_resource<memory_kind::device, cuda_vm_resource> pool;
+
+    vector<std::thread> threads;
+
+    for (int t = 0; t < 10; t++) {
+      threads.push_back(std::thread([&]() {
+        AsyncPoolTest(pool, blocks, mtx, stream);
+      }));
+    }
+    for (auto &t : threads)
+      t.join();
+  }
```

Comment:
Is there any value in showing stats like in `TEST(MMAsyncPool, MultiThreadedSingleStreamRandom) {`?

Reply:
I'd have to add them to async_pool (it currently doesn't have any). The global_pool_ member is not accessible here.